server/db/migration/public/migrations/20180709092931-create-user-table.js

```diff
@@ -19,7 +19,7 @@ exports.setup = function(options, seedLink) {
 };
 
 exports.up = function(db) {
-  var filePath = path.join(__dirname, 'sqls', '20180709092931-create-user-sessions-table-up.sql');
+  var filePath = path.join(__dirname, 'sqls', '20180709092931-create-user-table-up.sql');
```

Comment:
it might be good to rename the js migration file as well

Reply:
It's renamed, for some reason it doesn't show here

Reply:
It was renamed, for some reason it doesn't show here